CMakeLists.txt

```diff
@@ -353,6 +355,8 @@ set(plugins
   gazebo_catapult_plugin
   gazebo_usv_dynamics_plugin
   gazebo_parachute_plugin
+  gazebo_buoyancy_plugin
+  gazebo_lta_aerodynamics_plugin
```

Comment:
Would it be possible to maybe make the buoyancy plugin and aerodynamics plugin a single plugin? or is there a reason that these have to be separate?